spec/features/admin_update_family_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+feature "Admin updates a family" do
+  scenario "successfully" do
+    family = create(:family)
+
+    visit edit_family_path(family)
+
+    fill_in "family_last_name", with: "New Last Name"
+    fill_in "family_capability", with: 5
```

Comment:
This line is not needed, if you are not also checking for it later in the spec.

Reply:
Don in fixup ```6a8e803```.